website/docs/addon_maya_ornatrix_admin.md

```diff
@@ -0,0 +1,59 @@
+---
+id: addon_maya_ornatrix_admin
+title: Ornatrix for Maya
+sidebar_label: Ornatrix
+description: AYON Maya Ornatrix Admin documentation
+toc_max_heading_level: 5
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+{versions.Maya_Badge}
+</ReactMarkdown>
+
+
+## Maya Addon Setting for Ornatrix Maya
+
+### Creators
+- `ayon+settings://maya/create/CreateOxRig`
+- `ayon+settings://maya/create/CreateOxCache`
+
+![](assets/maya/admin/ornatrix_creators.png)
+
+### Validators
+
+- `ayon+settings://maya/publish/ValidateOrnatrixRigContent`
+- `ayon+settings://maya/publish/ValidateOrnatrixCacheContent`
+
+![](assets/maya/admin/ornatrix_validators.png)
+
+### Loaders 
+
+
+#### Ornatrix Rig Loader
+
+Access the settings for the Ornatrix Rig Loader at:
+- `ayon+settings://maya/load/OxRigLoader`
+
+![](assets/maya/admin/ornatrix_loaders.png)
+
+1. Toggle the loader on or off.
+2. **Create Ornatrix Cache instance on load:**
+    Automatically create an Ornatrix Cache instance upon loading, akin to how an animation rig automatically generates an animation instance for publishing loaded character rigs—applied specifically for Ornatrix rigs.
```

Comment:
Not sure the more concise to way to elaborate this, but.
```suggestion
    Ornatrix Cache instance is automatically created whenever an Ornatrix rig is loaded into the scene, akin to how the loaded character rig automatically generates the animation instance for publishing.
```

Reply:
me neither, let's summon @BigRoy .

Reply:
what about: 
```suggestion
    When enabled, an Ornatrix Cache instance is automatically created each time an Ornatrix rig is loaded into the scene, similar to how loading a character rig automatically creates the animation instance for publishing.
```